frame/elections-phragmen/src/lib.rs

```diff
@@ -973,18 +973,29 @@ impl<T: Trait> Module<T> {
 			);
 			T::ChangeMembers::set_prime(prime);
 
-			// outgoing members lose their bond.
-			let mut to_burn_bond = outgoing.to_vec();
+			// outgoing members who are no longer a runner-up lose their bond.
+			let mut to_burn_bond = outgoing
+				.iter()
+				.filter(|o| new_runners_up_ids_sorted.binary_search(o).is_err())
+				.cloned()
+				.collect::<Vec<_>>();
```

Comment:
looks like this will need re-benchmarking, right?

Reply:
Same as the other one, I think the change is insubstantial, but let's see